src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateDeconstructMethodTests.cs

```diff
@@ -335,6 +335,32 @@ internal void Deconstruct(out int x, out int y)
     }
 }");
         }
+        [Fact]
+        public async Task TestDeconstructionPositionalPattern()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void Method()
+    {
+        if(this is C(""[||])) { }
```

Comment:
`""` in a verbatim string is just a single double quote.

```suggestion
        if(this is C(""""[||])) { }
```